SkietbaanBE/SkietbaanBE/Controllers/NotificationController.cs

```diff
@@ -19,46 +19,41 @@ public NotificationController(ModelsContext context)
             _context = context;
         }
 
-        // GET: api/Notification
-        [HttpGet]
-        public IEnumerable<Notifications> Get()
+        [HttpGet("{id}", Name = "GetNotificationById")]
+        public async Task<Notifications> GetNotificationById(int id)
         {
-            var notificationQuery = from noti in _context.Notifications
-                                   where noti.IsRead == false
-                                   select noti;
-            List<Notifications> notificationList = notificationQuery.ToList<Notifications>();
-            return notificationList;
+            var notification = _context.Notifications.SingleOrDefault(bright => bright.Id == id);
```

Comment:
variable name is bright? anyways if it fails to fetch anything what happens?

Reply:
Returns NotFound